topics/climate/tutorials/cmip6/tutorial.md

```diff
@@ -84,34 +84,23 @@ CMIP6 is the Coupled Model Intercomparison Project in its 6th phase. [CMIP6](htt
 * Calculate an ensemble mean of all used models
 * Calculate and plot the seasonal mean 
   
-## Get data
+## Starting Galaxy Climate JupyterLab
 
-> ### {% icon hands_on %} Hands-on: Data upload
+ ## {% icon hands_on %} Hands-on: Launch JupyterLab for Ocean / Atmosphere / Land / Climate Python ecosystem in Galaxy
 >
-> 1. Create a new history for this tutorial
-> 2. Import the files from [Zenodo]() or from the shared data library
->
->    ```
->    
->    ```
->    ***TODO***: *Add the files by the ones on Zenodo here (if not added)*
->
->    ***TODO***: *Remove the useless files (if added)*
->
->    {% snippet faqs/galaxy/datasets_import_via_link.md %}
->    {% snippet faqs/galaxy/datasets_import_from_data_library.md %}
->
-> 3. Rename the datasets
-> 4. Check that the datatype
->
->    {% snippet faqs/galaxy/datasets_change_datatype.md datatype="netcdf" %}
->
-> 5. Add to each database a tag corresponding to ...
->
->    {% snippet faqs/galaxy/datasets_add_tag.md %}
+> Currently JupyterLab for Ocean / Atmosphere / Land / Climate Python ecosystem in Galaxy is available on [Live.useGalaxy.eu](https://live.usegalaxy.eu) only. JupyterLab for Ocean / Atmosphere / Land / Climate Python ecosystem and not the default JupyterLab in Galaxy contains all the python packages and additional software we need for running Earth System Model, including Functionally Assembled Terrestrial Ecosystem Simulator (FATES). The default JupyterLab in Galaxy would not be sufficient for executing all the tasks in th
```

Comment:
instead of:

for running Earth System Model, including Functionally Assembled Terrestrial Ecosystem Simulator (FATES)

we can replace by:

for analyzing and visualizing Pangeo CMIP6 climate data.